cfn-resources/cloud-backup-schedule/.rpdk-config

```diff
@@ -2,19 +2,20 @@
     "artifact_type": "RESOURCE",
     "typeName": "MongoDB::Atlas::CloudBackupSchedule",
     "language": "go",
-    "runtime": "go1.x",
+    "runtime": "provided.al2",
     "entrypoint": "bootstrap",
     "testEntrypoint": "bootstrap",
     "settings": {
         "version": false,
         "subparser_name": null,
         "verbose": 0,
         "force": false,
-        "type_name": null,
```

Comment:
is this type_name = null happenning in other resources?

Reply:
In general our .rpdk-config file structure is quite inconsistent across all the resources - might need some documentation update of our wiki (or just make a PR change to fix them all)